ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- Remove text and sections that do not apply -->
+
+This issue tracker is only for technical issues related to zcashd.
+
+General zcash questions and/or support requests and are best directed to the [Zcash Forums](https://forum.z.cash) or [Community Rocket.Chat](https://chat.zcashcommunity.com).
+
+For reporting security vulnerabilities or for sensitive discussions with our security team, please contact [security@z.cash](mailto:security@z.cash). You can use the [GPG key](https://z.cash/gpg-pubkeys/security.asc) (fingerprint: `AF85 0445 546C 18B7 86F9  2C62 88FB 8B86 D8B5 A68C`) to send an encrypted message. The key and fingerprint are duplicated on our [Public Keys page](https://z.cash/support/pubkeys.html).
+
+### Describe the issue
+
```

Comment:
Add one or two more newlines here, and maybe a placeholder (e.g. `[ISSUE DESCRIPTION]`) to show that text should be entered here.